doc/tntcxx_getting_started.rst

```diff
@@ -0,0 +1,524 @@
+      ``git``, a version control system
+   * `install <https://linuxize.com/post/how-to-unzip-files-in-linux/#installing-unzip>`_
+      the ``unzip`` utility
+   * `install <https://gcc.gnu.org/install/>`_ the ``gcc`` compiler `complied
+      with the C++17 standard <https://gcc.gnu.org/projects/cxx-status.html#cxx17>`_.
+   * `install <https://cmake.org/install/>`_ the ``cmake`` and ``make`` tools.
+
+#. If you don't have Tarantool in you OS, install it in one of the ways:
+
+   * from a package--refer to `OS-specific instructions <https://www.tarantool.io/en/download/>`_
+   * from the `source <https://www.tarantool.io/en/download/os-installation/building-from-source/>`_.
+
+#. Clone the Tarantool C++ connector repository and build the connector from
+   the source.
+
+   .. code-block:: bash
+
+      git clone git@github.com:tarantool/tntcxx.git
+      cd tntcxx
+      cmake .
+      make
+
+.. _gs_cxx_prereq_tnt_run:
+
+Starting Tarantool and creating a database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Start Tarantool :ref:`locally <getting_started-using_package_manager>`
+or :ref:`in Docker <getting_started-using_docker>`
+and create a space with the following schema and index:
+
+.. code-block:: lua
+
+    box.cfg{listen = 3301}
+    t = box.schema.space.create('t')
+    t:format({
+             {name = 'id', type = 'unsigned'},
+             {name = 'a', type = 'string'},
+             {name = 'b', type = 'number'}
+             })
+    t:create_index('primary', {
+             type = 'hash',
+             parts = {'id'}
+             })
+
+.. IMPORTANT::
+
+   Please do not close the terminal window where Tarantool is running.
+   You will need it later to connect to Tarantool from our C++ application.
+
+.. _gs_cxx_prereq_access:
+
+Setting up access rights
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To be able to execute the necessary operations in Tarantool, we need to grant
+the ``guest`` user with the read-write rights. The simplest way is to grant
+the user with the :ref:`super role <authentication-roles>`:
+
+.. code-block:: lua
+
+   box.schema.user.grant('guest', 'super')
+
+.. _gs_cxx_connect:
+
+Connecting to Tarantool
+-----------------------
+
+There are three main parts of the C++ connector: the IO-zero-copy buffer,
+the msgpack encoder/decoder, and the client that handles requests.
+
+To set up connection to a Tarantool instance from our C++ application, we need
+to specify the following in the application code:
+
+* :ref:`to embed the connector into the application <gs_cxx_connect_embed>`
+* :ref:`to instantiate a connector client and a connection object <gs_cxx_connect_instantiate>`
+* :ref:`to define connection parameters and invoke the method to connect <gs_cxx_connect_connecting>`
+* :ref:`to define error handling behavior <gs_cxx_connect_error>`.
+
+.. _gs_cxx_connect_embed:
+
+Embedding connector
+~~~~~~~~~~~~~~~~~~~
+
+The connector is embedded in a C++ application by including the main header:
```

Comment:
Passive voice hides the real action here.

```suggestion
Embed the connector in your C++ application by including the main header:
```

Reply:
Corrected.